x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -231,7 +231,7 @@ static RoleDescriptor kibanaSystem(String name) {
                 RoleDescriptor.IndicesPrivileges.builder().indices("logs-fleet_server*").privileges("read", "delete_index").build(),
                 // Fleet creates and writes this index for sync integrations feature
                 RoleDescriptor.IndicesPrivileges.builder()
-                    .indices("fleet-synced-integrations")
+                    .indices("fleet-synced-integrations*")
```

Comment:
Can we make this more explicit? It'll be a good reference for the future if we ever need to understand the change. 

```suggestion
                    .indices("fleet-synced-integrations", "fleet-synced-integrations-ccr*")
```